gradle/libs.versions.toml

```diff
@@ -26,8 +27,10 @@ kotlin-logging = { group = "io.github.oshai", name = "kotlin-logging", version =
 kotlin-logging-microutils = { group = "io.github.microutils", name = "kotlin-logging", version = "3.0.5" }
 kotlin-html = "org.jetbrains.kotlinx:kotlinx-html:0.11.0"
 kotlin-coroutines-core = { group = "org.jetbrains.kotlinx", name = "kotlinx-coroutines-core", version.ref = "kotlinCoroutines" }
+kotlin-serialization-json = { group = "org.jetbrains.kotlinx", name = "kotlinx-serialization-json", version.ref = "kotlinSerializationJson" }
 
 ktor-client-core = { group = "io.ktor", name = "ktor-client-core", version.ref = "ktor" }
 
 zt-zip = { group = "org.zeroturnaround", name = "zt-zip", version = "1.17" }
 logback-classic = { group = "ch.qos.logback", name = "logback-classic", version = "1.5.6" }
+jackson-dataformat-xml = { group = "com.fasterxml.jackson.dataformat", name = "jackson-dataformat-xml", version = "2.17.1" }
```

Comment:
Is `jackson-dataformat-xml` used?

Reply:
fixed in 68fa9ad499e56e66d84e58ab26c29184112b4241